multiversx_sdk_cli/cli_output.py

```diff
@@ -69,7 +69,7 @@ def build(self) -> dict[str, Any]:
             output["contractAddress"] = contract_address
 
         if self.transaction_on_network:
-            transaction_on_network_dict = transaction_on_network_to_dictionary(self.transaction_on_network)
+            transaction_on_network_dict = self.transaction_on_network.raw
```

Comment:
Good, now we return the data as it comes from the API (no change - except of the field omissions below) :+1: